apps/gateway-ui/src/GatewayApi.ts

```diff
@@ -72,7 +72,7 @@ export class GatewayApi implements ApiInterface {
       });
 
       if (res.ok) {
-        const address: string = await res.text();
+        const address: string = await res.json();
```

Comment:
Sure this introduces a bug. the `address` api returns a plain text instead of a json. I know, it's quite odd :/